runtime/crab/src/lib.rs

```diff
@@ -936,6 +947,56 @@ impl darwinia_crab_issuing::Trait for Runtime {
 	type WeightInfo = ();
 }
 
+/// Fixed gas price of `1`.
+pub struct FixedGasPrice;
+impl FeeCalculator for FixedGasPrice {
+	fn min_gas_price() -> U256 {
+		// Gas price is always one token per gas.
+		0.into()
+	}
+}
+
+parameter_types! {
+	pub const ChainId: u64 = 43;
```

Comment:
Need to check this ChainId again.

Reply:
Have no idea, are there any convention or registry guiding this?

Reply:
@AurevoirXavier  Did you know something about this?